ab_tool/canvas.py

```diff
@@ -90,6 +92,19 @@ def _get_installed_intervention_point_urls(self):
         return installed_intervention_point_urls
 
 
+def get_unsorted_students(request, experiment):
+    request_context = get_canvas_request_context(request)
+    course_id = get_lti_param(request, "custom_canvas_course_id")
+    try:
+        enrollments = list_users_in_course_users(
+                request_context, course_id, None, enrollment_type="student").json()
+    except RequestException as exception:
```

Comment:
FYI: the SDK has it's own exception type now so you don't have to call on the underlying requests library.